demos/ros_examples/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ros_examples</name>
+  <version>0.0.0</version>
+  <description>Control libraries demos included in a ROS package</description>
+  <maintainer email="baptiste.busch@epfl.ch">root</maintainer>
+  <license>GPLv3</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>tf</build_depend>
+  <build_export_depend>roscpp</build_export_depend>
+  <build_export_depend>tf</build_export_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>tf2</exec_depend>
```

Comment:
`tf` or `tf2` ?

Reply:
good `tf` indeed. I tried initially with `tf2` but didn't work. To be honest I think that the syntax is actually `tf` but the underlying mechanism is `tf2`

Reply:
also, why not just use the `depend` tag that summarizes all three depends? http://wiki.ros.org/catkin/package.xml#Format_2_.28Recommended.29